cmd/cri-containerd/options/options.go

```diff
@@ -35,6 +35,17 @@ type ContainerdConfig struct {
 	ContainerdSnapshotter string `toml:"snapshotter"`
 	// ContainerdEndpoint is the containerd endpoint path.
 	ContainerdEndpoint string `toml:"endpoint"`
+	// DefaultRuntime is the runtime to use in containerd. We may support
+	// other runtimes in the future.
+	DefaultRuntime string `toml:"runtime"`
+	// DefaultRuntimeEngine is the name of the runtime engine used by containerd.
+	// We may support other runtime engines in the future.
+	DefaultRuntimeEngine string `toml:"runtime_engine"`
+	// DefaultRuntimeRoot is the directory used by containerd for runtime state.
+	DefaultRuntimeRoot string `toml:"runtime_root"`
+	// DefaultRuntimeSystemdCgroup is a flag for enabling systemd cgroup support
+	// in the selected runtime engine.
+	DefaultRuntimeSystemdCgroup bool `toml:"runtime_systemd_cgroup"`
```

Comment:
I'd prefer to remove the prefix "Default".    "Default" indicate only a preferd value. but It is a option  here,we can actually give many value.